portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,16 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['Last year was my first ever doing cheerleading', 'I am a native Spanish speaker', 'I once sprained both of my ankles while walking', 'I know how to speak and write some Mandarin'];
```

Comment:
Please format such that this line doesn't scroll horizontally, it helps for reviews.

Reply:
I believe I added a new commit to this branch that takes care of this. Was it done properly?